test/models/message_test.js

```diff
@@ -1,5 +1,5 @@
 const { expect } = require('chai');
```

Comment:
Last thing. Could you add also test cases with `x-parser-original-traits` in Message and Operation object? :)